clients/drcachesim/CMakeLists.txt

```diff
@@ -171,6 +171,7 @@ target_link_libraries(drmemtrace_invariant_checker drdecode)
 
 configure_DynamoRIO_standalone(drmemtrace_opcode_mix)
```

Comment:
In the description:

> Fixes the corner case of the wrong previous instruction by using last_instr_in_cur_context_ instead of prev_instr_ when dealing with kernel xfers.

What line # has that change?  I don't see it.